### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pyle**
Updated section on using nginx proxy -
> container nginx .conf location was incorrect.
> container nginx port mapping was incorrect
The changes were made to work with existing transmission container

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ This is because the VPN is active, and since docker is running in a different ip
 There are several ways to fix this. You can pipe and do fancy iptables or ip route configurations on the host and in the Docker image. But I found that the simplest solution is just to proxy my traffic. Start a Nginx container like this:
 
 ```
-$ docker run -d -v /path/to/nginx.conf:/etc/nginx.conf:ro -p 8080:80 nginx
+$ docker run -d -v /path/to/nginx.conf:/etc/nginx/nginx.conf:ro -p 8080:8080 nginx
```
Comment (haugene) [inline README.md]: You are right about the path, but my example config listed below listens on port 8080. So update with port 8080 and I merge.

**Comment by pyle** [comment]
@haugene Made revisions to the Readme RE: nginx

**Comment by haugene** [comment]
Awwww, you were completely right the first time. It's supposed to be 8080:8080. I don't know what I was thinking. Can you revert your last commit? Sorry about the back and forth.

**Comment by pyle** [comment]
@haugene All fixed. 
I also changed the readme with out testing the new theory. All should be right now.

Cheers for the awesome container, works a treat

**Pull request merged**